web-app/js/portal/prototypes/NonCollapsingAccordionLayout.js

```diff
@@ -30,7 +29,16 @@ Ext.ux.NonCollapsingAccordionLayout = Ext.extend( Ext.layout.Accordion, {
     beforeExpandPanel : function(panel) {
         var panelToCollapse = this.currentlyExpandedPanel;  // A holder for the previously selected panel
         this.currentlyExpandedPanel = panel;                // Set the new panel as the currently expanded one
-        panelToCollapse.collapse();                         // Collapse the previously selected panel
+
+        if (panelToCollapse.collapsed) {
+            // fixes #1823
+            setTimeout(function() {
+                panelToCollapse.collapse();
+            }, 400 );
+        }
+        else {
+            panelToCollapse.collapse();                     // Collapse the previously selected panel
```

Comment:
I would say this comment is pretty redundant and that it can be removed. If you disagree (which you might) I think we should at least get rid of the spaces and also make a similar comment on line 36 (as it is essentially the same code, uncommented).